doc/cli/npm-logout.md

```diff
@@ -3,7 +3,7 @@ npm-logout(1) -- Log out of the registry
 
 ## SYNOPSIS
 
-    npm logout [--registry=url] [--scope=@orgname]
+    npm logout [--registry=<url>] [--scope=@<scope>]
```

Comment:
`--scope=<@scope>` – this is a point of some contention within the npm team, as @zeke can attest, but my feeling is that the `@` is part of the scope name.